src/tests/jsonLogic.test.js

```diff
@@ -344,5 +349,72 @@ describe('jsonLogic: cross-values validations', () => {
       expect(fieldB.maximum).toEqual(8);
       expect(fieldB.statement).toEqual({ description: 'Must be bigger than 4 and smaller than 8' });
     });
+
```

Comment:
> [@johnstonbl01] This one looks good to me. Will defer to Sandrina on her previous comments.

We still are on Part 3, but please _never ever_ let my comments (and long OOO/sick/moving) be your bottleneck.

If you don't trust your gut to approve this, trust the automated tests. For real, the tests are one of the things I am most proud of about this library. They cover a lot, and we also have Integration tests (Storybook+React). They are what give me full confidence that this is working and won't break previous versions.

In other words: _trust the tests, not me._ ✌️